tests/publiccloud/install_ipa.pm

```diff
@@ -0,0 +1,70 @@
+# SUSE's openQA tests
+#
+# Copyright © 2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Install IPA tool
+#
+# Maintainer: Clemens Famulla-Conrad <cfamullaconrad@suse.de>
+
+use base "opensusebasetest";
+use strict;
+use testapi;
+use utils;
+use registration 'add_suseconnect_product';
+use serial_terminal 'select_virtio_console';
+use version_utils qw(is_sle is_opensuse);
+
+
+sub run {
+    my ($self) = @_;
+
+    select_virtio_console();
+
+    if (is_sle) {
+        my $modver = get_required_var('VERSION') =~ s/-SP\d+//gr;
+        add_suseconnect_product('sle-module-public-cloud', $modver);
+    }
+    zypper_call('ar ' . get_required_var('PUBLIC_CLOUD_TOOLS_REPO'));
+    zypper_call('--gpg-auto-import-keys in python3-ipa python3-ipa-tests');
```

Comment:
1) If I am correct, this is to be run on SLE and OpenSuse, right?  Is PUBLIC_CLOUD_TOOLS_REPO valid for both SLE and OpenSuse? 
2) if you don't care about the output that zypper provides, you can use --quiet or -q

Reply:
The tools-repo depends on the distro+version. That is why I added it as test parameter.
The test on osd are based on a HDD which is currently a SLE12sp3.